### DISCUSSION TITLE
Video stream on Display

### DISCUSSION THREAD

**Discussion by Uli1900** [HASwitchPlate/openHASP#759] [Q&A]
Hi guys good job! openHASP make display usage much easier!

I did not find the feature to play a video on a screen (or a window/part of the screen)?

For example a RTSP stream?
With authentication (Example): rtsp://benutzername:Passwort@192.168.x.x:554/11
Without authentication (Example): rtsp://192.168.x.x:554/11

Would that be possible?
Thanks!

**Comment by fvanroie** [comment]
> I did not find the feature to play a video on a screen (or a window/part of the screen)?

That is not possible.

**Comment by Uli1900** [comment]
I want to show an live stream from an webserver (webcam). Is that possible?

**Reply by fvanroie** [reply]
Video playback or streaming is not implemented, you will have to add this feature yourself:
https://www.openhasp.com/0.7.0/firmware/compiling/custom/
